### DISCUSSION TITLE
Can't enable CAD Sketcher

### DISCUSSION THREAD

**Discussion by klgc2** [hlorus/CAD_Sketcher#380] [Feature Requests]
I saw a post on this a while back, but can't find it anymore. I'm wondering what's going on with this issue.
I downloaded 0.27.1 and installed it, but enabling fails with a traceback of folder designations, and "Exception: Shader Compile Error."
I don't know what console the traceback says there is more information in, there is nothing in my system Console.app.

I'm using Blender 3.5.0 on an intel MacBook Pro with macOS Monterey (ver: 12.6.4).

**Comment by figgelz** [comment]
I get the same error :(

**Comment by figgelz** [comment]
Changing Edit>Preferences>System>GPU Backend to OpenGL fixed it for me.
